src/Pulsar.Client/Internal/HttpLookupService.fs

```diff
@@ -140,11 +130,10 @@ type internal HttpLookupService (config: PulsarClientConfiguration, _connectionP
                     match isPersistent with
                     | true -> "PERSISTENT"
                     | false -> "NON_PERSISTENT"
-                let! response =
+                let! brokerResponse =
                     randomServiceUri.AbsoluteUri + $"admin/v2/namespaces/%s{ns.ToString()}/topics?mode=%s{mode}"
-                    |> httpClient.GetStreamAsync
+                    |> pulsarHttpClient.Get<string seq>
```

Comment:
Can you please change seq to array (in the interface as well)

Reply:
I change `HttpLookupService.fs` `ILookupService` and `BinaryLookupService` `GetTopicsUnderNamespace()` return type. I think this function only used in `PulsarClient.fs.GetTopicsByPattern()` so that we don't need to change caller strategy.